packages/components/tests/dummy/app/styles/pages/db-gutters.scss

```diff
@@ -0,0 +1,47 @@
+// GUTTERS
+
+// Loose spacing scale, move out to centralize for re-use:
+// (See: https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?node-id=2008%3A3)
+:root {
+    --hds-spacing-xxs: 4px;
+    --hds-spacing-xs: 8px;
+    --hds-spacing-sm: 12px;
+    --hds-spacing-md: 16px;
+    --hds-spacing-lg: 24px;
+    --hds-spacing-xl: 36px;
+    --hds-spacing-xxl: 48px;
+}
```

Comment:
The intent was **not** to make then generic design tokens. If we define them these way (and if we centralize them) then we revert this decision, that I would prefer not to (unless there's strong evidence about this need).